sherlock.py

```diff
@@ -40,41 +49,56 @@ def main():
         'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:55.0) Gecko/20100101 Firefox/55.0'
     }
 
+    # Create futures for all requests
+    for social_network in data:
+        url = data[social_network]['url'].format(username)
+
+        # This future starts running the request in a new thread, doesn't block the main thread
+        future = session.get(url=url, headers=headers)
+
+        # Store future in data for access later
+        data[social_network]['request'] = future
+
+    # Print results
     for social_network in data:
-        url = data.get(social_network).get("url").format(username)
-        error_type = data.get(social_network).get("errorType")
-    
 
-        r = requests.get(url, headers=headers)
-    
+        url = data[social_network]['url'].format(username)
+        error_type = data[social_network]['errorType']
+
+        # Retrieve future and ensure it has finished
+        future = data[social_network]['request']
+        response = future.result()
+
+        # Print result
         if error_type == "message":
-            error = data.get(social_network).get("errorMsg")
-            
```

Comment:
There were some issues with tabs and spaces being used together in this file. This isn't good, as python is very picky on tabs vs spaces. I converted all tabs to spaces to fix issues!